pkg/controller/backup_controller.go

```diff
@@ -385,3 +406,11 @@ func closeAndRemoveFile(file *os.File, log logrus.FieldLogger) {
 		log.WithError(err).WithField("file", file.Name()).Error("error removing file")
 	}
 }
+
+func getBackupScheduleName(backup *api.Backup) string {
+	scheduleName := kubeutil.LabelValue(backup, "ark-schedule")
+	if scheduleName == "" {
+		scheduleName = "UNSCHEDULED"
```

Comment:
If a Backup doesn't have a Schedule, we should simply not add the `schedule` label to the metrics rather than adding an arbitrary value..  This will simplify the prometheus queries later on.

Reply:
I think this allows users to collect metrics for all unscheduled backups easily. I don't have a strong preference on this though.

Reply:
How I've seen this work with Prometheus/Grafana is to use label matching operators:

`ark_server_backup_attempt_total{ark-schedule=".+"}` --> Matches any backup with a schedule specified
`ark_server_backup_attempt_total{ark-schedule=~"${Schedule}"` --> matches any backup with schedule using a variable, defaults to ".+" for "All", ignores any backups without a schedule
`ark_server_backup_attempt_total{ark-schedule!=".+"}` --> Matches all backups without a schedule